include/experimental/__p1673_bits/blas1_linalg_swap.hpp

```diff
@@ -127,15 +127,17 @@ struct is_custom_vector_swap_elements_avail<
 
 } // end anonymous namespace
 
-template<class ElementType_x,
-         extents<>::size_type ... ext_x,
+MDSPAN_TEMPLATE_REQUIRES(
+         class ElementType_x,
          class Layout_x,
          class Accessor_x,
          class ElementType_y,
-         extents<>::size_type ... ext_y,
          class Layout_y,
-         class Accessor_y>
-  requires (sizeof...(ext_x) == sizeof...(ext_y))
+         class Accessor_y,
+         extents<>::size_type ... ext_x,
```

Comment:
@youyu3 Just curious why you changed the order of template parameters here and in a few other places -- did the macro not work otherwise?  Would it help to surround the macro argument with parentheses, like this: `(extents<>::size_type ... ext_x)`

Reply:
Aha, with C++17, it didn't compile before I switched to the macro, with an error that says parameter packs should be at the end of the parameter list.  I just tested and confirmed that the original order works with the macro.  I'll back these out.

Reply:
Backed them out.